### ISSUE TITLE
A11y_ML .NET Model Builder_ Scenario_ High Contrast: Keyboard focus is not visible on the selected tab item in high contrast black and high contrast white mode

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1147]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #NonBenchmark; #Visual Studio; #High Contrast; #WCAG4.3.1;

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 Preview 6.0

### Repro Steps:

1.Enable High contrast mode in the system
2. Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Tab till "Scenario" tab item and verify whether keyboard focus is clearly visible on the control

### Actual Result:
 Keyboard focus is not visible on the selected tab item in high contrast black and high contrast white mode

### Expected Result:
Keyboard focus should be clearly visible on the tab items in high contrast black and high contrast white modes

### User Impact:
users who use high contrast mode to use the application will not get to know where the keyboard focus is if keyboard focus is not visible on the controls

### Attachment for Reference:

[1147_ML .NET Model Builder_ keyboard focus is not visible on tab items on high contrast.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5656774/1147_ML.NET.Model.Builder_.keyboard.focus.is.not.visible.on.tab.items.on.high.contrast.pptx)

**Comment by zewditu** [comment]
It is fixed  either by other work item or updated model Model Builder

**Comment by SaisreeDandamudi** [comment]
@zewditu @beccamc 
Verified in the latest build and the issue still re-pro. 
Please find the below attachment for reference
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0

[1147_ML.NET.Model.Builder_.keyboard.focus.is.not.visible.on.tab.items.on.high.contrast.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/6041781/1147_ML.NET.Model.Builder_.keyboard.focus.is.not.visible.on.tab.items.on.high.contrast.pptx)

**Comment by beccamc** [comment]
Build https://devdiv.visualstudio.com/_apis/resources/Containers/7343454/VSSetup?itemPath=VSSetup%2FMicrosoft.ML.ModelBuilder.vsix

**Comment by SaisreeDandamudi** [comment]
Verified the bug in latest build and the issue is fixed. Hence closing the bug
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.10.0 Preview 1.0

https://user-images.githubusercontent.com/51435580/111151515-8e6cc580-85b5-11eb-85be-75681722e593.mp4

**Comment by poojapatil36** [comment]
Github tag: #Closed
